docs/verifiable-document/issuing-document.md

```diff
@@ -8,12 +8,55 @@ After wrapping the documents and obtaining a merkle root, the documents are read
 
 In this guide, we will make use of a web application to issue the documents.
 
-## Prerequisites
+## Issuing Methods
+
+Currently, we provide two ways of issuing documents onto Ethereum:
+
+- [TradeTrust Admin Portal](https://admin.tradetrust.io/) or [Opencerts Admin Portal](https://admin.opencerts.io/) , a web app
+- [Open Attestation CLI](https://github.com/Open-Attestation/open-attestation-cli), a command line tool
+
+They are both decentralized apps (Dapps) as both enable you to interact with smart contracts on the Ethereum blockchain.
+
+## Using Open Attestation CLI
+
+### Prerequisites
+
+- [OpenAttestation CLI](../../component/open-attestation-cli) installed
```

Comment:
link doesn't work here too, should remove the additional ../

Reply:
done